tests/flows/native_token_bridge.go

```diff
@@ -56,6 +58,27 @@ func NativeTokenBridge(network interfaces.LocalNetwork) {
 	Expect(err).Should(BeNil())
 	log.Info("Native Token Bridge Contract Address: " + bridgeContractAddress.Hex())
 
+	{
+		// The deployer is used to also send native transfers so we don't have to create an additional address.
+		// The deployer will send 1.25*initialReserveImbalance over the course of the test, plus 10 eth to
+		// deploy the contract, plus a little bit of gas, so transfer 2*initialReserveImbalance to be safe.
```

Comment:
```suggestion
		// The deployer is used to also send native transfers so we don't have to create an additional address.
		// The deployer will send 1.25*initialReserveImbalance over the course of the test. Send 2*initialReserveImbalance native tokens
		// so that it also has enough to cover transaction fees, including deploying the contract. 
```